### ISSUE TITLE
Support for <details> <summary>

### ISSUE THREAD

**Issue by Airkro** [simov/markdown-viewer#16]
How about add support for `<details> <summary>` to make a dropdown?

like this:
<details>
<summary>test</summary>
Hello World !
</details>

**Comment by simov** [comment]
This currently works:

```
<details>
<summary>Hello World</summary>
Something here ...
</details>
```

But you need the `sanitize` option disabled.

There are two problems though:

1. The mouse cursor doesn't change when you hover over the the *summary*
2. Only inline tags are rendered inside the *details* block, meaning that you can't have a fenced code blocks inside the *details* tag

So I'm going to look at those two issues and let you know once I have something ready.

**Comment by simov** [comment]
I just published version 2.8 where you can choose between different compilers.

For full support for `<details> <summary>` tags you have to choose the `showdown` compiler.

Let me know if you have any other issues.

**Comment by simov** [comment]
Another quick release, I switched to the [Remark](https://github.com/wooorm/remark) compiler. You should receive the update automatically. It fixes this issue.
